improver_tests/calibration/reliability_calibration/test_ApplyReliabilityCalibration.py

```diff
@@ -269,6 +277,22 @@ def test_insufficient_forecast_count(self):
         self.assertIsNone(result[0])
         self.assertIsNone(result[1])
 
+    def test_incomplete_bins_allowed(self):
+        """Test that if the forecast count is insufficient in a single bin, but
+        the minimum_bin_fraction is set to allow this, the function returns the
+        expected values."""
+
+        expected = (
+            np.array([0.0, 0.25, 0.5, 0.75, 10.0]),
+            np.array([0.0, 0.0, 0.25, 0.5, 7.5]),
+        )
+
+        self.reliability_cube.data[0, 2, -1] = 100
```

Comment:
Perhaps this should be `self.reliability_cube.data[0, :, -1] = 100` given that currently this creates an inconsistent reliability table cube? The expected results of `10.0` for the forecast probability and `7.5` for the observation frequency couldn't actually occur within the reliability table, so it would be better to use expected values that are achievable using a consistent reliability table.

Reply:
Yep. I wasn't actually thinking about what the value were meant to be, so thanks for the correction.